src/main/java/cms/gov/madie/measure/services/MeasureService.java

```diff
@@ -36,6 +44,57 @@ public class MeasureService {
   private final MeasureRepository measureRepository;
   private final ElmTranslatorClient elmTranslatorClient;
   private final MeasureUtil measureUtil;
+  private final ActionLogService actionLogService;
+  private final TerminologyValidationService terminologyValidationService;
+
+  public Measure createMeasure(Measure measure, final String username, String accessToken) {
+    log.info("User [{}] is attempting to create a new measure", username);
+    checkDuplicateCqlLibraryName(measure.getCqlLibraryName());
+    validateMeasurementPeriod(
+        measure.getMeasurementPeriodStart(), measure.getMeasurementPeriodEnd());
+    Measure measureCopy = measure.toBuilder().build();
+    Set<MeasureErrorType> errorTypes = new HashSet<>();
+    try {
+      measureCopy = updateElm(measure, accessToken);
```

Comment:
Since `updateElm` will return the passed in `measure`, need to pass `measureCopy` to prevent it from being pointed to the original `measure`.